core/src/main/scala/org/bykn/bosatsu/Declaration.scala

```diff
@@ -440,10 +450,10 @@ object Declaration {
     }
   }
 
-  def ifElseP(expr: Indy[Declaration]): Indy[IfElse] = {
+  def ifElseP(arg: P[NonBinding], expr: Indy[Declaration]): Indy[IfElse] = {
 
-    def ifelif(str: String): Indy[(Declaration, OptIndent[Declaration])] =
-      Indy.block(Indy.lift(P(str ~ spaces ~ maybeSpace)) *> expr, expr)
+    def ifelif(str: String): Indy[(NonBinding, OptIndent[Declaration])] =
+      Indy.block(Indy.lift(P(str) ~ spaces ~ maybeSpace ~ arg), expr)
```

Comment:
why `spaces ~ maybeSpace`? What is going on there? Seems like `P(str ~ spaces ~ arg)` would do it.